### PR TITLE
[Android] Cleanup of unnecessary imports in java packages

### PR DESCRIPTION
**Pull request by joseluismarti**
## Description
Remove unneeded imports from java files in ```tools/android/packaging/xbmc/src/ ```

Ordering import statements in ```Main.java.in``` and ```Splash.java.in```

## How has this been tested?
Compiles without errors. Run several tests on Android devices.

## Types of change
- [ ] **Bug fix** (non-breaking change which fixes an issue)
- [X] **Clean up** (non-breaking change which removes non-working, unmaintained functionality)
- [ ] **Improvement** (non-breaking change which improves existing functionality)
- [ ] **New feature** (non-breaking change which adds functionality)
- [ ] **Breaking change** (fix or feature that will cause existing functionality to change)
- [ ] **Cosmetic change** (non-breaking change that doesn't touch code)
- [ ] **None of the above** (please explain below)

## Checklist:
- [X] My code follows the **[Code Guidelines](https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md)** of this project 
- [ ] My change requires a change to the documentation, either Doxygen or wiki
- [ ] I have updated the documentation accordingly
- [X] I have read the **[Contributing](https://github.com/xbmc/xbmc/blob/master/docs/CONTRIBUTING.md)** document
- [ ] I have added tests to cover my change
- [ ] All new and existing tests passed

### CODE REVIEW

**Review by ksooo — APPROVED** [review]
Thank you. Nice cleanup.

**Comment by fuzzard** [comment]
Just do that in a separate commit and then add to this PR

File: tools/android/packaging/xbmc/src/Splash.java.in [hunk 1]
```diff
@@ -1,52 +1,51 @@
...
 import android.Manifest;
-import android.net.Uri;
-import android.os.AsyncTask;
-import android.os.Build;
-import android.os.Bundle;
 import android.app.Activity;
 import android.app.ActivityManager;
 import android.app.ActivityManager.RunningTaskInfo;
 import android.app.AlertDialog;
+import android.content.BroadcastReceiver;
```
[the remark this answers is not in the document]
Comment (joseluismarti) [reply tools/android/packaging/xbmc/src/Splash.java.in:10]: In **Java**, import statements are always divided in groups:

- android
- androidx
- com
- ...
- java

with each group separated by a single blank line.

**Within a group there are no blank lines**, and the imported names appear in ASCII sort order.

[the remark this answers is not in the document]
Comment (howie-f) [reply tools/android/packaging/xbmc/src/Splash.java.in:10]: ok, thanks for explaining and sorry for noise

**Review by wsnipex — APPROVED** [review]

**Review by enen92 — APPROVED** [review]
Awesome, thanks for the cleanup

**Review by howie-f — APPROVED** [review]

**Pull request merged**